sdk/features/unit/dummyfiles/empties/empty_annotations2.py

```diff
@@ -0,0 +1,14 @@
+import unittest
+
+from features.lusid_feature import lusid_feature
+
+
+class Empties2Tests(unittest.TestCase):
```

Comment:
why is this needed ie. what's different vs `EmptiesTests`?

Reply:
Again to test that it targets all files, and that doesn't throw any errors in the event of some files being unannotated